src/components/ContactLinks/ContactLinks.scss

```diff
@@ -0,0 +1,35 @@
+@import "styles/colors.scss";
+
+.contact-links {
+  .title {
+    font-size: 2.5em;
+    font-weight: 350;
+  }
+
+  .text {
+    font-size: 1.5em;
+    font-weight: 300;
+  }
+
+  .textIcon {
+    font-size: 1.5em;
+    margin-right: 0.2em;
```

Comment:
Should be done in tailwind, missed this on first look.

Reply:
resolved